draft-ietf-taps-interface.md

```diff
@@ -2099,10 +2099,10 @@ Properties will include different information:
 
 ## Generic Connection Properties {#connection-props}
 
-The Connection Properties defined as independent of the chosen protocol stack,
-and available on all Connections are defined in the subsections below.
+Generic Connection Properties are defined independent of the chosen protocol stack
+and therefor are available on all Connections.
```

Comment:
s/therefor/therefore
and: I'm really unsure about the "are" after "therefore". It might be correct, but I'm slightly more sure that the sentence is ok *without* the "are", i.e.: "Generic Connection Properties are defined independent of the chosen protocol stack and therefore available on all Connections."